### ISSUE TITLE
light channel mapping

### ISSUE THREAD

**Issue by YifanC** [DUNE/larnd-sim#182]
Currently, we are only simulating with vanilla light channel mapping.
Is there any benefit for us to introduce an option for more realistic mapping in larnd-sim? It seems that the downstream ndlar-flow is already capable of handling this for data.
The mapping itself may depend much on the ADC setup which is not static.

**Comment by liviocali** [comment]
I agree. Since the light DAQ for FSD/ND is not fixed at all, I would stay with the channel scheme as it is in larndsim. That's the most generic.
